.github/workflows/update-contributors.yaml

```
@@ -0,0 +1,18 @@
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  update-contributors:
+    runs-on: ubuntu-latest
+    name: Atualiza CONTRIBUTORS
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/okfn-contributors
```

Reparei agora no nome do diretório da action.

O "okfn" na verdade se refere à [Open Knowledge Foundation](https://okfn.org/en/) (organização internacional). Enquanto "okbr" se refere à [Open Knowledge Brasil](https://ok.org.br/) (capítulo nacional).

Pensei em pedir pra mudar para `okbr-contributors` mas também vejo como opções aqui `qd-contributors`, `querido-diario-contributors` ou simplesmente `contributors`.

O que acha?
```suggestion
      - uses: ./.github/actions/contributors
```